plugins/inputs/intel_powerstat/intel_powerstat.go

```diff
@@ -41,6 +41,7 @@ const (
 type PowerStat struct {
 	CPUMetrics     []string        `toml:"cpu_metrics"`
 	PackageMetrics []string        `toml:"package_metrics"`
+	ReadTimeout    int64           `toml:"msr_read_timeout"`
```

Comment:
Please make this field an actual duration:
```suggestion
	ReadTimeout    config.Duration `toml:"msr_read_timeout"`
```

Reply:
That would be good.